### PR TITLE
Prepare announcement of Coq 8.10.0

### PR DESCRIPTION
**Pull request by vbgl**
*Do not merge before the release actually happen.*

### CODE REVIEW

File: incl/macros.html [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+<#def CURRENTVERSIONDATE>October 2019</#def>
+<#def CURRENTVERSIONTAG>V8.10.0</#def>
 <#def CURRENTVERSIONURL><#BASEURL><#CURRENTVERSIONTAG></#def>
 <#def CURRENTCREDITSURL>https://github.com/coq/coq/blob/<#CURRENTVERSIONTAG>/CREDITS</#def>
 
-<#def UPCOMINGVERSION>8.10</#def>
-<#def UPCOMINGVERSIONURL>https://github.com/coq/coq/releases/tag/V8.10%2Bbeta3</#def>
+<#ifdef UPCOMINGVERSION>
+<#def UPCOMINGVERSION>8.11</#def>
+<#def UPCOMINGVERSIONURL>https://github.com/coq/coq/releases/tag/V8.11%2Bbeta1</#def>
```
Comment (Zimmi48) [inline incl/macros.html]: Without any change to the main page, this is going to introduce broken and inaccurate information.

Comment (vbgl) [reply incl/macros.html]: What change do you want in the main page? “Don’t read the source code of this website as it may contain inaccurate data.”

Comment (vbgl) [reply incl/macros.html]: Moreover I fail to see anything inaccurate there.

Comment (Zimmi48) [reply incl/macros.html]: Sorry for the noise. I should have compiled the website before making any comment. I had forgotten about the recent enhancements by Guillaume that made this possible.

**Comment by vbgl** [comment]
May I merge this?

**Comment by Zimmi48** [comment]
Yes!

**Pull request merged**